### PR TITLE
Day 4: Styling changes

### PR DESCRIPTION
**Pull request by benjp009**
# 👋 Micronaut Reviewer 

## Bookstore: styling

[See the Pull Request Objectives](https://github.com/microverseinc/curriculum-react-redux/blob/main/bookstore/styling.md)

Screenshot 2022-01-20 at 09 30 13

### Project requirements

- [x] You will find all the details of the Bookstore design in Zeplin.
- [x] Login to Zeplin (hint: you will find the credentials in the sneak peek of the project).
- [x] Open the project Bookstore CMS.
- [x] Use the information detailed in Zeplin to style the website.
- [x] Your final design should match Zeplin's design.

### Pair Programming 🎳

For this week project, there is no PP required. 

Happy review :)

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -15,17 +15,21 @@ export default function App() {
...
+            <ul className="navBar">
+              <li>
+                <Link to="/" className="navItem">Books</Link>
+              </li>
+              <li>
+                <Link to="categories" className="navItem">Categories</Link>
+              </li>
+            </ul>
+          </nav>
+        </div>
```
Comment (TedLivist) [inline src/App.js]: - [x] Your app is expected to match the design of the Zeplin template. Kindly add the user icon to the right on the Navbar.

Your Implementation | Zeplin Design
:---:|:---:
benjp-icon-marked | benjp-icon

File: src/components/Book.js [hunk 2]
```diff
@@ -16,14 +16,21 @@ const Book = ({
   return (
     <>
       <div className="oneBook">
```
Comment (TedLivist) [inline src/components/Book.js:18]: - [x] Your book is missing the demarcating line as provided in the template. Kindly add it.

Your Implementation | Zeplin Design
:---:|:---:
benjp-demarc | benjp-orig

**Review by TedLivist — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ ♻️

Hi @benjp009,

Good job so far!
✔️ Good documentation
✔️ Nicely modularized code
However, there are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
- [x] To keep your codebase neat, kindly delete the `src/components/BookTitle.js` file since it is only comprised of commented code. Commented code make our project look messy.

  benjp-book

- [x] _Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by hiromataba — APPROVED** [review]
# Status: Approved :tada:  :champagne:  :confetti_ball: 
Hi @benjp009    :wave: , 

appr


Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

### To Highlight :1st_place_medal: 

- No Linter errors :heavy_check_mark: 
- Your README file looks very professional :+1: 
- Awesome design :heavy_check_mark: 


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**